### DISCUSSION TITLE
September Newsletter

### DISCUSSION THREAD

**Discussion by briancl2** [briancl2/CustomerNewsletter#6] [Customer Newsletter]
This is a personally curated newsletter for just my customers. The purpose is to pull together and share current resources, announcements, and events. I welcome your feedback, and please let me know if you would like to add or remove anyone from this list.

### Copilot
This is the most significant single month of Copilot changes I’ve seen:
* We shipped [Copilot in ](https://github.blog/news-insights/product-news/github-copilot-now-available-in-github-com-for-copilot-individual-and-copilot-business-plans/)[github.com](http://github.com/) which integrates Copilot directly into the github.com experience to [chat with repositories,](https://docs.github.com/en/enterprise-cloud@latest/copilot/using-github-copilot/asking-github-copilot-questions-in-githubcom#asking-exploratory-questions-about-a-repository) ask about [common support scenarios](https://github.blog/changelog/2024-09-10-copilot-chat-in-github-com-is-now-aware-of-common-support-scenarios-and-githubs-documentation), include [GHAS alert context](https://github.blog/changelog/2024-09-05-copilot-chat-in-github-com-is-now-contextually-aware-of-github-advanced-security-alerts) in chats, and [auto-summarize pull requests](https://docs.github.com/en/enterprise-cloud@latest/copilot/using-github-copilot/creating-a-pull-request-summary-with-github-copilot). This feature is currently in [beta](https://docs.github.com/site-policy/github-terms/github-pre-release-license-terms) and disabled by default.
* As a reminder from the [revised plan](https://github.blog/changelog/2024-08-06-revised-release-plan-for-copilot-subscription-based-network-routing/) for Copilot Subscription-based network routing, please validate your [network firewall settings](https://docs.github.com/en/copilot/managing-copilot/managing-github-copilot-in-your-organization/configuring-your-proxy-server-or-firewall-for-copilot) prior to the Oct 31 breaking change taking effect. This change will allow enterprises to block employees that try to use their unauthorized Copilot Individual license instead of a Copilot Business license which adheres to enterprise policies. 
* OpenAI released their new [o1](https://github.blog/news-insights/product-news/openai-o1-in-github-copilot/) model, and GitHub shipped [early access](https://github.blog/changelog/2024-09-19-sign-up-for-openai-o1-access-on-github/) to o1-preview and o1-mini for Copilot Chat in VS Code.
* [Code Referencing is now GA](https://github.blog/news-insights/product-news/code-referencing-now-generally-available-in-github-copilot-and-with-microsoft-azure-ai/), so customers can disable public code blocking without losing indemnification.
* [Copilot Extensions is now in public beta](https://github.blog/news-insights/product-news/enhancing-the-github-copilot-ecosystem-with-copilot-extensions-now-in-public-beta/) available to all customers who wish to opt-in.
* [Fine-tuned models are now in private beta](https://github.blog/news-insights/product-news/fine-tuned-models-are-now-in-limited-public-beta-for-github-copilot-enterprise/) for Copilot Enterprise customers.
* [Modernizing COBOL with GitHub Copilot](https://resources.github.com/software-development/modernizing-cobol-with-github-copilot/) whitepaper published by our very own Mashrur Hossain.
* [Larger context window, improved test generation, and define custom instructions](https://github.blog/changelog/2024-09-09-larger-context-window-improved-test-generation-and-more-in-vs-codes-copilot-chat/) in VS Code’s Copilot Chat.

### Miscellaneous
* GitHub Platform update: Customers will begin transitioning to our [enhanced billing platform](https://github.blog/changelog/2024-09-24-enhanced-billing-platform-for-enterprises/) that allows for more fine-grained control and visibility of costs and usage across products, cost centers, and more. This also allows for customers to attach and charge to multiple Azure subscriptions for metered usage.
* Industry News: [Nicole Forsgren explains SPACE](https://www.youtube.com/watch?v=O2rbekHpG4Q), how it came about, and how the framework can be applied to any technology - not just developer productivity - such as cloud adoption or automated testing. In the video, she creates example metrics in realtime to explain how to use SPACE to develop a measuring framework.

### Recent Recordings
* Sept 11 - [GitHub Copilot Prompt Engineering 101](https://developer.microsoft.com/en-us/reactor/events/23470/)
* Sept 18 - [Build and Deploy Applications to Azure using GitHub Actions](https://www.youtube.com/watch?v=sVhN92fNZso)
* Sept 23 - [Get your on-prem apps AI-ready using GitHub Copilot and Azure Migrate Code Assessment](https://www.youtube.com/watch?v=UW4_xpiI6TA)
* Brian’s personally curated Copilot Training & Tips [Youtube playlist](https://www.youtube.com/playlist?list=PLCiDM8_DsPQ1WJ5Ss3e0Lsw8EaijUL_6D), updated monthly

### Upcoming Webinars
* Oct 1 - [End-to-End AI application deployments with GitHub Copilot](https://developer.microsoft.com/en-us/reactor/events/23594/)
* Oct 3 - [Extending GitHub Copilot to chat with your Postgres Database](https://developer.microsoft.com/en-us/reactor/events/23595/)
* Oct 8 - [10x Developer with GitHub Copilot](https://developer.microsoft.com/en-us/reactor/events/23597/)
* Oct 8 - [Build powerful READMEs with Markdown](https://developer.microsoft.com/en-us/reactor/events/23578/)
* Oct 9 - [Mastering Copilot: Streamlining Your Development Process with Swimm, Crowdbotics, Pinecone, & Faros](https://resources.github.com/events/webinar-mastering-github-with-copilot-oct9/)
* Oct 10 - [Remediation with GitHub Copilot Autofix](https://resources.github.com/global-security-webinar-oct24/)
* Oct 15 - [Create a website using GitHub Copilot and Codespaces](https://developer.microsoft.com/en-us/reactor/events/23579/)
* Oct 17 - [An Introduction to GitHub Copilot & What’s New](https://msit.events.teams.microsoft.com/event/4d7f196d-edab-4997-8c12-0bae1dcd4182@72f988bf-86f1-41af-91ab-2d7cd011db47)
* Oct 22 - [Automate your repository using GitHub Actions](https://developer.microsoft.com/en-us/reactor/events/23581/)
* Oct 22 - [Learn to Code using GitHub Copilot and Codespaces](https://developer.microsoft.com/en-us/reactor/events/23630/)
* Oct 23 - [Data Residency on GitHub](https://resources.github.com/data-residency-webinar-emea/)
* Nov 5 - [GitHub Copilot plans and features](https://developer.microsoft.com/en-us/reactor/events/23718/)
* Nov 12 - [Developer Use Cases for AI with GitHub Copilot](https://developer.microsoft.com/en-us/reactor/events/23719/)
* Nov 19 - [Testing with GitHub Copilot](https://developer.microsoft.com/en-us/reactor/events/23722/)
* Dec 12 - [Prompt Engineering 300](https://developer.microsoft.com/en-us/reactor/events/23687/)

### Upcoming Events
* Chicago - Oct 8-9: Reuters Total Health - Transforming Healthcare with Generative AI: Leveraging GitHub Copilot for Secure Innovation
* Minneapolis - Oct 9: CyberFest Oktoberfest with Endor Labs and GitHub
* Chicago - Oct 24: [GOTO Chicago AI Days](https://gotochgo.com/2024)
* San Francisco - Oct 29-30: [GitHub Universe](https://githubuniverse.com/#tickets)

**Comment by tomswilliams** [comment]
Wow 🎉 🎉 🎉 Curation at its finest 🙇

**Comment by Nickismyname** [comment]
Holy cow. @briancl2 this is sweet. 
#subscribe

**Comment by RaduButa3** [comment]
🔥 @briancl2 .  Thank you for curating and sharing!
